modules/core/multihop/proof_gen.go

```diff
@@ -35,26 +37,58 @@ type Endpoint interface {
 	Counterparty() Endpoint
 }
 
+func EndpointToString(endpoint Endpoint) string {
+	chainHeight := endpoint.GetChainHeight()
+	kvHeight := endpoint.GetKeyValueProofHeight()
+	consHeight := endpoint.GetConsensusHeight()
+	return fmt.Sprintf("{\"chain_id\": %q, \"client_id\": %q, \"connection_id\": %q, \"chain_height\": %d, \"kv_proof_height\": %q, \"consensus_height\": %q}",
+		endpoint.ChainID(), endpoint.ClientID(), endpoint.ConnectionID(), chainHeight, kvHeight, consHeight)
+}
+
```

Comment:
Do you mind if we drop this func too? The ibc-go module user should come up with their own debugging format.